### ISSUE TITLE
Put item behaviour of Localstack DDB different from AWS DDB local

### ISSUE THREAD

**Issue by posilva** [localstack/localstack#2121]
Hi notice a different behaviour between Localstack DDB and AWS Local DDB when executing a Put Item operation:
The default return_values implemented in DDB and DDB Local is None and LocalStack DDB is returning the all_new, this is breaking my tests. Additionally I added explicit ReturnValues=None and still returns ALL_NEW

Steps to Reproduce:
- Install AWS Local db docker image and run it in port 8000
- Install Localstack following readme 
- Run both 

1. Create table against Localstack: 
`aws --endpoint-url=http://localhost:4569 dynamodb create-table --table-name test_table  --attribute-definitions AttributeName=first,AttributeType=S AttributeName=second,AttributeType=N --key-schema AttributeName=first,KeyType=HASH AttributeName=second,KeyType=RANGE --provisioned-throughput ReadCapacityUnits=5,WriteCapacityUnits=5`

2. Put Item against Localstack
`aws --endpoint-url=http://localhost:4569 dynamodb put-item --table-name test_table  --item '{"first":{"S":"Jack"},"second":{"N":"42"}}'
`
The record was returned after put_item command

Execute the same steps 1 and 2 changing the endpoint port to 8000, put-item does not return the values and assumes the default behaviour ReturnValues=None as stated in the documentation (https://docs.aws.amazon.com/cli/latest/reference/dynamodb/put-item.html): 

> If ReturnValues is not specified, or if its value is NONE , then nothing is returned. (This setting is the default for ReturnValues .)

**Comment by posilva** [comment]
Maybe is fixed on https://github.com/localstack/localstack/pull/2103
Related item https://github.com/localstack/localstack/issues/2049

**Comment by posilva** [comment]
I tested with the master branch and this issue is not fixed for the use case of this ticket

**Comment by whummer** [comment]
Thanks for reporting @posilva . We believe that this should be fixed in #2298 . Can you please give it a try with the `latest` version? Please report here if the problem persists. Thanks

**Comment by gruckion** [comment]
@posilva was this fixed? What version of localstack was you using? I am having the same issue with.

```
version: '3.9'

services:
  dynamodb-local:
    image: localstack/localstack:0.12.12
    ports:
      - '4566:4566'
    environment:
      - SERVICES=dynamodb,serverless,lambda,ssm
      - DATA_DIR=/tmp/localstack/data
      - DOCKER_HOST=unix:///var/run/docker.sock
      - DEBUG=1
    volumes:
      - "/var/run/docker.sock:/var/run/docker.sock"

volumes:
  dynamodb-data:

```
